CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# Release 2.29.0 30-06-2021
+
+- Update API version to v10 (#547)
```

Comment:
remove line 3,  it is not useful for espv2 users.

Reply:
Done.